### PR TITLE
EDUCATOR-5231: Change wording in ORA "Manage Teams" Instructor view under "Team's Final Grade" when Team Submission is Cancelled

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** Show the right message in the staff area student lookup when a user is part of a team submission that has been cancelled 

**What changed?**

- This was partially fixed by EDUCATOR-5185
- Look up workflow cancellation by team submission workflow's "canonical" submission's individual submission uuid

**Developer Checklist**
- [x] Reviewed the [release process](./release_process.md)
- [x] Translations up to date
- [x] JS minified, SASS compiled
- [x] Test suites passing on Jenkins
- [x] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

JIRA: [EDUCATOR-5231](https://openedx.atlassian.net/browse/EDUCATOR-5231)

FIY: @edx/masters-devs-gta

### CODE REVIEW

File: openassessment/templates/openassessmentblock/staff_area/oa_student_info.html [hunk 1]
```diff
@@ -169,7 +169,7 @@ <h2 class="staff-info__title ui-staff__subtitle">
                     {% if not is_team_assignment %}
                         <p>{% trans "The learner's submission has been removed from peer assessment. The learner receives a grade of zero unless you delete the learner's state for the problem to allow them to resubmit a response." %}</p>
                     {% else %}
-                        <p>{% trans "The teams's submission has been removed from peer assessment. The team receives a grade of zero unless you delete the team's state for the problem to allow them to resubmit a response." %}</p>
+                        <p>{% trans "The team’s submission has been removed from grading. The team receives a grade of zero unless you delete the a team member’s state for the problem to allow the team to resubmit a response" %}</p>
```
Comment (Andytr1) [inline openassessment/templates/openassessmentblock/staff_area/oa_student_info.html:172]: Stylistic question: are we omitting the period at the end of sentence?

Comment (jansenk) [reply openassessment/templates/openassessmentblock/staff_area/oa_student_info.html:172]: Not intentionally, I just copy and pasted this. I'll add it.
